routers/router.mad1.yml

```
@@ -9,3 +9,13 @@
     remote_address: vpn.diogopassos.pt
     remote_port: 22097
     public_key: czJoGDj8855EvO0nwqVLH1Bv/I6quSaD1mddFE408nE=
+- name: ARFNET
+  asn: AS4242423269
```

```suggestion
  asn: 4242423269
```

Only need to specify the number.